src/Drivers/CustomDriver.php

```diff
@@ -8,20 +8,25 @@
 
 class CustomDriver extends SlackDriver
 {
+    private const MESSAGE = 'message_action';
+    private const REPORT = 'report_message';
+    private const EVENT = 'event_callback';
+    private const MEMBER_JOIN = 'member_joined_channel';
```

Comment:
I like the idea of having constants instead of strings. however I have a couple of concerns about these specific changes:

1. These constants are only used in this file, and they are never used more than once each, hence the refactor seems unnecessary.
2. The constant names are not obvious. If we are keeping them, I'd prefer something more meaningful like `message_action` → `PAYLOAD_TYPE_MESSAGE_ACTION`, or `member_joined_channel` → `EVENT_TYPE_ MEMBER_JOINED_CHANNEL`.